pages/about.js

```diff
@@ -1,10 +1,18 @@
 // import Header from "../comps/Header";
-import Layout from "../components/MyLayout";
+import Layout from "../comps/MyLayout";
```

Comment:
I didn't notice this before, but usually the file name matches the thing it exports, so `MyLayout.js` should export a `MyLayout` component or should be renamed just `Layout.js`